src/inputs/Button/Button.tsx

```diff
@@ -2,17 +2,22 @@ import { PropsWithChildren } from "preact/compat";
 import { h } from "preact";
 
 import * as styles from "./Button.module.css";
+import { Plus, Spinner } from "../../assets"
+
 
 interface ButtonProps {
   onClick(): void;
```

Comment:
**issue:**  Button is missing one, optional property - `type`, which should be of value 'submit' or 'button'